mozilla/foundation.mozilla.org
# now scrolling to section after being pushed down by new data

Closes #7069

**Link to test page**: https://mozfest-foundation-s-pulse-list-0odhfg.mofostaging.net/en/tickets/#mozfest-2019-healthy-ai

**Steps to test**:

1. Please visit the test page above
2. Scroll down to find the different years of mozfest
3. Click on one of the years, it should take you to the same year section on the page.
4. It should render in the correctly location even after new data loads in, or, should scroll to the correct position after.
